src/main/views/courts/tabs/addressesContent.njk

```diff
@@ -0,0 +1,178 @@
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+<h2 class="govuk-heading-l">Addresses</h2>
+
+{% if errors|length %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+{% if updated === true %}
+  {{ govukPanel({
+    titleText: "Addresses updated"
+  }) }}
+{% endif %}
+
+<div class="govuk-body">
+  <p>A court can have up to 2 addresses. Only the second address can be removed.<p>
+</div>
+
+<h3 class="govuk-heading-m">Primary Address</h3>
+
+<input type="hidden" name="writeToUsTypeId" value={{writeToUsTypeId}}></input>
```

Comment:
I  think it was mentioned on the last PR to try and avoid hidden inputs if we can? Is this one unavoidable? Might be on a similar train of thought to the slug for the court; best to confirm

Reply:
We need this so that we can validate there is no duplication of visit us address types on the server.